code/modules/mining/machine_vending.dm

```diff
@@ -12,9 +12,12 @@
 	var/icon_deny = "mining-deny"
 	var/obj/item/card/id/inserted_id
 	var/list/prize_list = list( //if you add something to this, please, for the love of god, sort it by price/type. use tabs and not spaces.
-		new /datum/data/mining_equipment("30 Marker Beacons",			/obj/item/stack/marker_beacon/thirty,								100),
+		new /datum/data/mining_equipment("1 Marker Beacon",				/obj/item/stack/marker_beacon,										5),
+		new /datum/data/mining_equipment("10 Marker Beacons",			/obj/item/stack/marker_beacon/ten,									500),
+		new /datum/data/mining_equipment("30 Marker Beacons",			/obj/item/stack/marker_beacon/thirty,								150),
 		new /datum/data/mining_equipment("Shelter Capsule",				/obj/item/survivalcapsule,											400),
 		new /datum/data/mining_equipment("Regen. Core Stabilizer",		/obj/item/hivelordstabilizer,										400),
+		new /datum/data/mining_equipment("Skeleton Key",				/obj/item/skeleton_key,												777),
```

Comment:
make it consistent with the cost of other items. 1000 would be good

Reply:
Amount is relatively arbitrary, but somewhere between 500 and 1000 makes it a requirement but not something too annoying to do

Reply:
@bloons3 What I mean is, keep it a multiple of 100 or at least a multiple of 50

Reply:
I concur, it should be a multiple of 50 or 100. No funny numbers.

Reply:
750